File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.java
class Solution {
    public int solution(int[] num_list) {
        int numEven = 0;
        int numOdd = 0;
        for(int i = 0; i < num_list.length; i++){
            if((i+1) % 2 == 0) numEven += num_list[i];
            else numOdd += num_list[i];
        }
        return numEven >= numOdd ? numEven : numOdd;
    }
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    ret1, ret2 = 0, 0
    for i,x in enumerate(num_list):
        if (i + 1) % 2 == 1:
            ret1 += x
        else: ret2 += x
    return ret1 if ret1 >= ret2 else ret2